admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -150,15 +162,23 @@ main() {
         effect: NoSchedule'
     fi
 
+    if [ -x "$(command -v envsubst)" ]; then
+        ENVSUBST="$(command -v envsubst) < $TEMPLATE_PATH"
+    elif [ -x "$(command -v docker)" ]; then
+        ENVSUBST="docker run --rm -v $TEMPLATE_PATH:$TEMPLATE_PATH -eTLS_PRIVATE_KEY -eTLS_CERTIFICATE -eCA_BUNDLE -eRBAC_ROLE_NAME -eNAME -eNAMESPACE -eIMAGE_NAME -eTOLERATIONS supinf/envsubst $TEMPLATE_PATH"
+    else
+        fatal_error "Unable to run envsubst"
+    fi
```

Comment:
This whole `if` block can be removed, I think?

Reply:
Yes, was an oversight on my part. Sorry for that.